.buildkite/fpm-pipeline.yml

```diff
@@ -5,6 +5,7 @@ env:
   IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
   DOCKER_REGISTRY: "docker.elastic.co"
   STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
+  GCP_DEFAULT_X86_64_MACHINE_TYPE: "n2-standard-16"
```

Comment:
I am ambivalent whether this big jump from the default `n1-standard-4` with ~15Gi to ~64Gi -- the motivation is based on findings from interactive runs that segfauls _almost_ get eliminated already by going to 32Gi (-8).

On the cost side, based on Buildkite stats we are averaging something like 1build/week for main, so I think it's not a big problem.

Reply:
If we move forward with a smaller instance and then need to trigger more rebuilds due to more failures, it might not get that much cheaper as the instance is running longer.
Maybe a smaller instance could be chosen if/when the base image gets upgrade to Ubuntu-24 as we were seeing less segfaults then.